frontend/packages/dev-console/src/components/monitoring/queries.ts

```diff
@@ -71,12 +71,21 @@ export const monitoringDashboardQueries: MonitoringQuery[] = [
     humanize: humanizeBinaryBytes,
     byteDataType: ByteDataTypes.BinaryBytes,
   },
+  {
+    query: _.template(
+      `topk(25, sort_desc(sum((kubelet_volume_stats_used_bytes {namespace='<%= namespace %>'}* on (namespace,persistentvolumeclaim) group_right() kube_pod_spec_volumes_persistentvolumeclaims_info)) by (namespace,pod)))`,
```

Comment:
as far as I know these metrics refer to persistent volumes, the title though states `Local Storage Usage`. Is this intended?

Reply:
ping @abhi-kn